spec/Section 3 -- Type System.md

```diff
@@ -1688,6 +1725,38 @@ input ExampleInputObject {
 | `{ b: $var }`            | `{ var: null }`         | Error: {b} must be non-null.         |
 | `{ b: 123, c: "xyz" }`   | `{}`                    | Error: Unexpected field {c}          |
 
+Following are examples of input coercion for a oneOf input object type with a
+`String` member field `a` and an `Int` member field `b`:
+
+```graphql example
+input ExampleInputTagged @oneOf {
+  a: String
+  b: Int
+}
+```
+
+| Literal Value            | Variables               | Coerced Value                                       |
+| ------------------------ | ----------------------- | --------------------------------------------------- |
+| `{ a: "abc", b: 123 }`   | `{}`                    | Error: Exactly one key must be specified            |
+| `{ a: null, b: 123 }`    | `{}`                    | Error: Exactly one key must be specified            |
+| `{ a: null }`            | `{}`                    | Error: Value for member field {a} must be non-null  |
+| `{ b: 123 }`             | `{}`                    | `{ b: 123 }`                                        |
+| `{ a: $var, b: 123 }`    | `{ var: null }`         | Error: Exactly one key must be specified            |
+| `{ a: $var, b: 123 }`    | `{}`                    | Error: Exactly one key must be specified            |
+| `{ b: $var }`            | `{ var: 123 }`          | `{ b: 123 }`                                        |
+| `$var`                   | `{ var: { b: 123 } }`   | `{ b: 123 }`                                        |
+| `"abc123"`               | `{}`                    | Error: Incorrect value                              |
+| `$var`                   | `{ var: "abc123" } }`   | Error: Incorrect value                              |
+| `{ a: "abc", b: "123" }` | `{}`                    | Error: Exactly one key must be specified            |
+| `{ b: "123" }`           | `{}`                    | Error: Incorrect value for member field {b}         |
+| `{ a: "abc" }`           | `{}`                    | `{ a: "abc" }`                                      |
+| `{ b: $var }`            | `{}`                    | Error: Value for member field {b} must be specified |
+| `$var`                   | `{ var: { a: "abc" } }` | `{ a: "abc" }`                                      |
```

Comment:
```suggestion
| `$var`                   | `{ var: { a: "abc" } }` | `{ a: "abc" }`                                      |
| `$var`                   | `{ var: {} }`           | Error: Exactly one key must be specified            |
| `$var`                   | `{ var: { a: "abc", b: 123 } }` | Error: Exactly one key must be specified    |
| `$var`                   | `{ var: { a: null } }`  | Error: Value for member field {a} must be non-null  |
```

Reply:
Assuming this matches the current proposal outlined in this PR, let's merge these additional samples to increase clarity.  If the PR changes, then we can update the samples to match.

Reply:
I've gone back through the previous recommendations too and have adopted many of them. I don't think we need to list every possible eventuality, e.g. if `{b:123}` works then `{a:"abc"}` should work - I don't think we need to list that.